homeassistant/components/geo_json_events/geo_location.py

```diff
@@ -104,7 +104,10 @@ async def async_update(self) -> None:
 
     def _update_from_feed(self, feed_entry: GenericFeedEntry) -> None:
         """Update the internal state from the provided feed entry."""
-        self._attr_name = feed_entry.title
+        if feed_entry.properties and feed_entry.properties.get("name"):
+            self._attr_name = feed_entry.properties.get("name")
+        else:
+            feed_entry.title
```

Comment:
Because this is not a statement ;)

Reply:
Oh my, that is so embarassing! I thought I fixed that but mustn't have hit push.